app/src/main/java/com/zulip/android/models/Message.java

```diff
@@ -533,5 +573,203 @@ public Drawable getDrawable(String source) {
                 source, null, null, parser, emojiGetter, app.getServerURI());
         return converter.convert();
     }
+    //endregion
+
+    //region model-getter-setters
+    public String getSubject() {
+        return subject;
+    }
+
+    public void setSubject(String subject) {
+        if (subject != null && subject.equals("")) {
+            // The empty string should be interpreted as "no topic"
+            // i18n here will be sad
+            this.subject = "(no topic)";
```

Comment:
Idk, i actually didn't write this code. I think It was just shifted from 300's to 500's (line number). I can change it to using Strings.xml no problem.